source/simulate/analyse-simulation.md

```diff
@@ -0,0 +1,92 @@
+# How to analyse a simulation
+
+When you run a simulation with the Python API, you might want to understand how the result was calculated and what intermediate variables and parameters has been taken into account.
+
+> To trace a simulation calculation with the Web API, please see [/trace endpoint documentation](openfisca-web-api/trace-simulation.md).
+
+
+## Call simulation tracer
+
+Let's say that you already have a `Simulation` object called `simulation` and, following [run a simulation](simulate/run-simulation.md) example, you tried to calculate the `housing_allowance` for a set of households.
```

Comment:
Hmm this phrasing may be misleading, because if you already have a `simulation` object that you used for calculations, you cannot activate the trace _a posteriori_ to get details about the calculation.

Reply:
I'd replace this paragraph and the following by something like:

> Let's suppose you ran a simulation, calculated the `housing_allowance` for a set of households, and would like to understand in details where the final results come from. 
> 
>  To use the tracer, you should activate the `trace` option with `simulation.trace = True` _before_ running any calculation with a `simulation` object. This will allow you to inspect calculation steps and print them with `simulation.tracer.print_computation_log()`.